### DISCUSSION TITLE
How did the Titanium Hyperloop works

### DISCUSSION THREAD

**Discussion by ghazalialshafi** [appcelerator/titanium_mobile#13963] [General]
I want to understand in deep with reference how did the Hyperloop actually works which part and code of Titanium SDK is responsible for it.
Thank-you

**Comment by m1ga** [comment]
Hi, as it is open-source you can look at the code https://github.com/tidev/hyperloop.next
Especially the Android files have lots of comments in it: https://github.com/tidev/hyperloop.next/blob/master/android/src/hyperloop/HyperloopUtil.java
